src/main/java/de/muenchen/isi/api/controller/AbfrageController.java

```diff
@@ -55,7 +59,7 @@ public class AbfrageController {
     private final AbfrageApiMapper abfrageApiMapper;
 
     @Transactional(readOnly = true)
-    @GetMapping("infrastruktur-abfragen")
+    @GetMapping("")
```

Comment:
Der leere String kann aus den `XxxMapping` entfernt werden, da in der Annotation das Attribut per default einen leeren String besitzt.

Kann bei allen Mapping-Annotationen die einen leeren String beinhalten angewendet werden.

Reply:
erledigt.